app/assets/stylesheets/mobile/mobile.css.scss

```diff
@@ -1166,3 +1166,14 @@ select#aspect_ids_ {
   }
 }
 
+// Styles for mobile profile of other users
+#user_aspects {
+  width: auto !important;
+  float: right;
+  margin: 0 10px 0 0;
+  padding: 3px;
+
+  &.hasconnection {
+    background-color: #C1ED94;
```

Comment:
Is a color in `colors.css.scss` which you could use here? If you aren't able to find one it would be great if you could add this color to the file.